components/atom/progressBar/src/ProgressBarLine/index.js

```diff
@@ -18,6 +18,7 @@ const ProgressBarLine = ({
   indicatorBottom,
   indicatorTotal,
   isAnimatedOnChange,
+  isBorderless,
```

Comment:
```suggestion
  isBorderless = false,
```

even if `undefined` is `falsy`, should we set a default value of `false`, perhaps?